doc/source/whatsnew/v1.0.0.rst

```diff
@@ -221,8 +221,9 @@ Other enhancements
 - DataFrame constructor preserve `ExtensionArray` dtype with `ExtensionArray` (:issue:`11363`)
 - :meth:`DataFrame.sort_values` and :meth:`Series.sort_values` have gained ``ignore_index`` keyword to be able to reset index after sorting (:issue:`30114`)
 - :meth:`DataFrame.to_markdown` and :meth:`Series.to_markdown` added (:issue:`11052`)
-
 - :meth:`DataFrame.drop_duplicates` has gained ``ignore_index`` keyword to reset index (:issue:`30114`)
+- The ``pandas.datetime`` submodule is now deprecated. Import ``datetime`` directly instead(:issue:`30296`)
```

Comment:
```suggestion
- The ``pandas.datetime`` submodule is now deprecated. Import ``datetime`` directly instead (:issue:`30296`)
```

`pandas.datetime` is `datetime.datetime`, so instead of saying import it directly, could be more correct to say import it from the datetime module.